docs/macros.rst

```diff
@@ -510,6 +512,10 @@ API
     :param header_classes: A string of classes to apply to the table header (e.g ``'thead-dark'``).
     :param responsive: Whether to enable/disable table responsiveness.
     :param responsive_class: The responsive class to apply to the table. Default is ``'table-responsive'``.
+    :param safe_columns: Tuple with columns names to render HTML safe using ``|safe``.
+                Has priority over ``urlize`` parameter. Default is ``None``.
```

Comment:
```suggestion
                Has priority over ``urlize_columns`` parameter. Default is ``None``.
                WARNING: Only use this for sanitized user data to prevent XSS attacks.
```

Reply:
Done in next commit